ocw/lib/gce.py

```diff
@@ -154,50 +98,26 @@ def cleanup_all(self):
             if "items" not in response:
                 break
             for image in response["items"]:
-                # creation:2019-11-04T14:23:06.372-08:00
-                # name:sles12-sp5-gce-x8664-0-9-1-byos-build1-56
-                m = self.parse_image_name(image["name"])
-                if m:
-                    images.append(
-                        Image(
-                            image["name"],
-                            flavor=m["key"],
-                            build=m["build"],
-                            date=parse(image["creationTimestamp"]),
+                if self.older_than(parse(image["creationTimestamp"])):
+                    if self.dry_run:
+                        self.log_info( "Deletion of image {} skipped due to dry run mode", image["name"])
+                    else:
+                        self.log_info("Delete image '{}'", image["name"])
+                        request = (
+                            self.compute_client()
+                            .images()
+                            .delete(project=self.__project, image=image["name"])
                         )
-                    )
-                    self.log_dbg(
-                        "Image {} is candidate for deletion with build {}",
-                        image["name"],
-                        m["build"],
-                    )
-                else:
-                    self.log_err("Unable to parse image name '{}'", image["name"])
+                        response = request.execute()
+                        if "error" in response:
+                            for e in response["error"]["errors"]:
+                                self.log_err(e["message"])
+                        if "warnings" in response:
```

Comment:
```suggestion
                        elif "warnings" in response:
```

Reply:
my understanding that there could be errors AND warnings at the same time so not a good idea